src/components/i18n/DopplerIntlProvider.tsx

```diff
@@ -1,32 +1,37 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { IntlProvider } from "react-intl";
 import { messages_en } from "./en";
 import { messages_es } from "./es";
 import { flattenMessages, sanitizeLanguageOrDefault } from "./utils";
+import { useAppSessionState } from "../AppSessionStateContext";
 
 const messages = {
   es: messages_es,
   en: messages_en,
 };
 
 interface DopplerIntlProviderProps {
-  locale: string;
   children: React.ReactNode;
 }
 
-export const DopplerIntlProvider = (props: DopplerIntlProviderProps) => {
-  const sanitizedLocale = sanitizeLanguageOrDefault(
-    props.locale,
-    Object.keys(messages)
-  );
+export const DopplerIntlProvider = ({ children }: DopplerIntlProviderProps) => {
+  const appSessionState = useAppSessionState();
+  const [locale, setLocale] = useState("en");
```

Comment:
In my opinion, since most of our clients speak Spanish, this should be the default value.

```suggestion
  const [locale, setLocale] = useState("es");
```

Reply:
👍🏻